docs/configuration/configuration.md

```diff
@@ -129,10 +129,19 @@ See below for provider specific options
 
 `oauth2-proxy` supports having multiple upstreams, and has the option to pass requests on to HTTP(S) servers or serve static files from the file system. HTTP and HTTPS upstreams are configured by providing a URL such as `http://127.0.0.1:8080/` for the upstream parameter, this will forward all authenticated requests to the upstream server. If you instead provide `http://127.0.0.1:8080/some/path/` then it will only be requests that start with `/some/path/` which are forwarded to the upstream.
 
-Static file paths are configured as a file:// URL. `file:///var/www/static/` will serve the files from that directory at `http://[oauth2-proxy url]/var/www/static/`, which may not be what you want. You can provide the path to where the files should be available by adding a fragment to the configured URL. The value of the fragment will then be used to specify which path the files are available at. `file:///var/www/static/#/static/` will ie. make `/var/www/static/` available at `http://[oauth2-proxy url]/static/`.
+Static file paths are configured as a file:// URL. `file:///var/www/static/` will serve the files from that directory at `http://[oauth2-proxy url]/var/www/static/`, which may not be what you want. You can provide the path to where the files should be available by adding a fragment to the configured URL (see below).
 
 Multiple upstreams can either be configured by supplying a comma separated list to the `-upstream` parameter, supplying the parameter multiple times or provinding a list in the [config file](#config-file). When multiple upstreams are used routing to them will be based on the path they are set up with.
 
+> NOTE: When defining upstream URLs, if it does not end with a slash ('/') it will match ONLY that path EXACTLY, which is probably not what you're expecting. i.e. `http://127.0.0.1:8080/some/path` will serve ONLY the exact URL `http://[oauth2-proxy url]/some/path` (not anything below it such as `http://[oauth2-proxy url]/some/path/some/file.html`). To match anything "below" that you must also include the trailing slash.
```

Comment:
This is existing behavior I encountered, not anything introduced by this PR. I'm simply clarifying the documentation since it is certainly pretty unexpected (I spent a good little while trying to figure out what was going on myself).